gammapy/background/models.py

```diff
@@ -71,3 +81,775 @@ def evaluate(self, x, y):
+        # (TTYPEi, TFORMi), instead of in between.
+        # https://github.com/gammapy/gammapy/issues/298
+
+        return tbhdu
+
+    def to_fits_image(self):
+        """Convert cube background model to image fits format.
+
+        Returns
+        -------
+        hdu : `~astropy.io.fits.ImageHDU`
+            image containing the bg cube
+        """
+        hdu = fits.ImageHDU(data=self.background.value)
+        # TODO: store units (of bg) somewhere in header??!!!!
+        # TODO: implement WCS object to be able to read the det coords -> done
+        # TODO: energy binning: store in HDU table like for SpectralCube class
+
+        # get WCS object
+        wcs = self.det_wcs
+
+        # Now, write out the WCS object as a FITS header
+        wcs_header = wcs.to_header()
+
+        # transfering header values
+        # need to copy necessary values one by one
+        # TODO: can this be simplified? just copy the header doesn't
+        #       work, since it overwrites important keywords (for
+        #       instance the ones for the energy axis)
+        hdu.header['CTYPE1'] = wcs_header['CTYPE1']
+        hdu.header['CTYPE2'] = wcs_header['CTYPE2']
+        hdu.header['CUNIT1'] = wcs_header['CUNIT1']
+        hdu.header['CUNIT2'] = wcs_header['CUNIT2']
+        hdu.header['CDELT1'] = wcs_header['CDELT1']
+        hdu.header['CDELT2'] = wcs_header['CDELT2']
+        hdu.header['CRPIX1'] = wcs_header['CRPIX1']
+        hdu.header['CRPIX2'] = wcs_header['CRPIX2']
+        hdu.header['CRVAL1'] = wcs_header['CRVAL1']
+        hdu.header['CRVAL2'] = wcs_header['CRVAL2']
+
+        return hdu
+
+    def write(self, outfile, format='bin_table', write_kwargs=None):
+        """Write cube background model to fits file.
+
+        Several output formats are accepted, depending on the value
+        of the `format` parameter:
+
+        * bin_table (default and preferred format): `~astropy.io.fits.BinTableHDU`
+        * image (alternative format): `~astropy.io.fits.ImageHDU`
+
+        Depending on the value of the `format` parameter, this
+        method calls either `~astropy.io.fits.BinTableHDU.writeto` or
+        `~astropy.io.fits.ImageHDU.writeto`, forwarding the
+        `write_kwargs` arguments.
+
+        Parameters
+        ----------
+        outfile : `~string`
+            name of file to write
+        format : `~string`, optional
+            format of the bg cube to write
+        write_kwargs : `~dict`, optional
+            extra arguments for the corresponding `io.fits` `writeto` method
+        """
+        if write_kwargs is None:
+            write_kwargs = dict()
+
+        if format == 'bin_table':
+            self.to_fits_bin_table().writeto(outfile, **write_kwargs)
+        elif format == 'image':
+            self.to_fits_image().writeto(outfile, **write_kwargs)
+        else:
+            raise ValueError("Got format {}, which is not understood."
```

Comment:
I've tried to consistently report invalid string options as 
```
ValueError('Invalid format {}'.format(format)
```
It's the same info, but for consistency I prefer this over having a different wording as you did here in some places.
And when I have time, also list the valid values, but that's not needed.